lib/text-class/test/unit/Data/Text/ClassSpec.hs

```diff
@@ -40,6 +52,11 @@ spec = do
             in fromText @Int "patate" === Left (TextDecodingError err)
         it "fromText . toText === pure"
             $ property $ \(i :: Int) -> (fromText . toText) i === pure i
+        it "fromText ~ fromTextMaybe" $
+            property $ \(Digits t) ->
+              classify (isNothing (fromTextMaybe @Int t)) "invalid" $
+              classify ((compare 0 <$> fromTextMaybe @Int t) == Just GT) "valid negative" $
+              toList (fromTextMaybe @Int t) === toList (fromText t)
```

Comment:
https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters
Also, one level of indent is 4 spaces, rather than 2.

Suggest:
```hs
        it "fromText ~ fromTextMaybe" $
            property $ \(Digits t) ->
                classify (isNothing (fromTextMaybe @Int t))
                    "invalid" $
                classify ((compare 0 <$> fromTextMaybe @Int t) == Just GT)
                    "valid negative" $
                toList (fromTextMaybe @Int t) === toList (fromText t)
```
(The above fits comfortably within the column limit, and also has correct indentation.)